hpcflow/sdk/helper/helper.py

```diff
@@ -146,6 +197,19 @@ def restart_helper(
     start_helper(app, timeout, timeout_check_interval, watch_interval, logger=logger)
 
 
+def strip_helper_PID(app):
```

Comment:
Could you rename this functions to `read_helper_args` and define a new complementary function `write_helper_args`? For `read_helper_args` could you return a `dict` where the values are already cast to floats? Since we've changed the information stored in the file, I think we should update our function names/behaviour to reflect that.